evergreen.yml

```diff
@@ -259,11 +259,11 @@ functions:
       aws_key: ${aws_key}
       aws_secret: ${aws_secret}
       local_file: ${short_name}.tgz
-      remote_file: ${project}/${short_name}/${short_name}-${build_variant}-patch_${build_id}.tgz
+      remote_file: ${project}/patches/${build_id}/${short_name}.tgz
       bucket: mciuploads
       permissions: public-read
       content_type: ${content_type|application/gzip}
-      display_name: "${display_name} - Patch ${build_id}"
+      display_name: "${display_name} Patch"
```

Comment:
Somehow I missed how verbose the build id is. I've switched to a subfolder and a simple name.